source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -6,56 +6,92 @@ keywords: modules, plugins, unsupported, drupal, wordpress
 
-- Background Process  
-**Issue**: The module allows for Drupal to perform "parallel" (asynchronous non-blocking mode) requests. However, there are a number of limitations working in a distributed environment and function correctly on the platform.
+<hr>
 
-- Backup & Migrate
+### Cache Expiration
+**Issue**: Unfortunately, there is no way to selectively purge the Varnish cache.
 
-- Boost
+**Solution**: You can set a `NO_CACHE` cookie or set `Cache-Control: no-cache,max-age=0` header to bypass Varnish.
 
-- Cache Expiration
 
-- Global Redirect  
+<hr>
+### Global Redirect  
  **Issue**: Too many redirects error when site is in maintenance mode.  
- **Solution**: Ensure that the "Frontpage Redirect Handler" is not ticked in the Global Redirect administration page.
 
-- HTTPRL  
+ **Solution**: Ensure that the "Frontpage Redirect Handler" is not ticked in the Global Redirect administration page.
+<hr>
+### HTTPRL  
 **Issue**: This module can severely impact performance. This may be the result of module code or its configuration on the platform that results in the spikes.
 
-- IMCE 6.x
+<hr>
+### IMCE 6.x
+**Issue**: Operations on directories containing an inordinate amount of files will likely hit the load balancer timeout threshold (30 seconds).
 
-- Mobile Tools
+**Solution**: The alternative for now is to break up the files into smaller groups.
 
-- Mollom  
- **Issue**: Cookies break Varnish.
 
-- Pathologic  
+<hr>
+### Mobile Tools
+**Issue**: Conflicts with Varnish. See [https://www.drupal.org/node/1976162#comment-7411366](https://www.drupal.org/node/1976162#comment-7411366).
+<hr>
+### Mollom  
+**Issue**: Conflicts with Varnish by returning the I.P. address of the proxy cache servers.
+<hr>
+### Pathologic  
  **Issue**: The path of the base URL is changed and cached by the module itself.  
- **Solution**: The [documentation on Drupal.org](https://drupal.org/node/257026) for the module mentions the issues and the remedy, which is a cache clear operation. If you are unable to exclude cached data from your dumps or avoid migrating cache data, you should clear your site’s cache after importing the data.
 
-- Registry Rebuild  
+ **Solution**: The [documentation on Drupal.org](https://drupal.org/node/257026) for the module mentions the issues and the remedy, which is a cache clear operation. If you are unable to exclude cached data from your dumps or avoid migrating cache data, you should clear your site’s cache after importing the data.
+<hr>
+### Registry Rebuild  
 This is built into the Pantheon platform.
 
-- Schema  
-**Issue**: The module doesn't work with the MySQL TIMESTAMP column type in our heartbeat table, which is part of how we maintain status around whether or not a site and its database is active. This is a [known bug](https://drupal.org/node/468644) in the schema module.  
+To rebuild the registry of a site, create a backup from your Dashboard, then run:
```

Comment:
Can we link to our existing documentation on how to rebuild registry rather than duplicating it here?